go-controller/pkg/ovn/pods.go

```diff
@@ -399,8 +407,8 @@ func (oc *Controller) addLogicalPort(pod *kapi.Pod) error {
 		if err != nil {
 			return err
 		}
-
-		marshalledAnnotation, err := util.MarshalPodAnnotation(&podAnnotation)
+		var marshalledAnnotation map[string]string
+		marshalledAnnotation, err = util.MarshalPodAnnotation(&podAnnotation)
 		if err != nil {
```

Comment:
@abhat I don't think this change is required because of named return? Thanks.

Reply:
it doesn't hurt though.

Reply:
ok, that is fine.